hil/rest.py

```diff
@@ -231,6 +235,10 @@ def _do_validation(schema, kwargs):
             raise validation_error
         final_kwargs[k] = kwargs[k]
 
+    print version
+    if version['version'] != VERSION:
```

Comment:
I haven't figured out why we cannot put the check for the version after the `schema.validate`

If we put it after, it will never check the version.